src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -88,6 +88,9 @@ public virtual bool IgnoresAccessibility
             get { return false; }
         }
 
+        [Experimental("RSEXPERIMENTAL001")]
+        public abstract bool DisableNullableAnalysis { get; }
```

Comment:
```suggestion
        public abstract bool DisablesNullableAnalysis { get; }
```

To be parallel with "IgnoresAccessibility"?

Reply:
I would prefer `NullableAnaysisIsDisabled`. `DisableNullableAnalysis` is an action (this property will disable nullable analysis when set), and `DisablesNullableAnalysis` implies that this property will do something to another component. Neither is the correct interpetation: nullable analysis is disabled on this semantic model.

Reply:
Works for me.